libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/BlurHashAsyncImage.kt

```diff
@@ -71,7 +71,7 @@ fun BlurHashAsyncImage(
 }
 
 @Composable
-fun BlurHashImage(
+private fun BlurHashImage(
```

Comment:
Are these visibility changes enforced by konsist too?

Reply:
Yes, else the file should be renamed to `BlurHashImage.kt`, which mean that actually this function should be moved to its own file).

The inner rule (of mine) is: one public composable per file.